xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -828,6 +828,30 @@ When submitting a certificate signing request via a web browser or other graphic
         <exception cref="T:System.ArgumentNullException">
           <paramref name="signatureGenerator" /> is <see langword="null" />.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">A cryptographic error occurs while creating the signing request.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains a <see langword="null" /> value.
```

Comment:
Actually, the `OtherRequestAttributes` API will also be available in RC2, and so the CI warning will go away when we bump the ref assemblies.